shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/test/java/org/apache/shardingsphere/proxy/initializer/impl/GovernanceBootstrapInitializerTest.java

```diff
@@ -227,4 +227,10 @@ public void assertDecorateTransactionContexts() {
     protected void prepareSpecifiedInitializer() {
         setInitializer(new GovernanceBootstrapInitializer());
     }
+
+    private StandardMetaDataContexts mockStandardMetaDataContexts() {
+        StandardMetaDataContexts standardMetaDataContexts = mock(StandardMetaDataContexts.class);
+        when(standardMetaDataContexts.getProps()).thenReturn(new ConfigurationProperties(new Properties()));
+        return standardMetaDataContexts;
+    }
```

Comment:
Another PR fix NPE, so, could you revert the changes?

Reply:
ok~